lib/roku-develop.coffee

```diff
@@ -245,6 +248,41 @@ module.exports        = RokuDevelop =
   toggle: ->
     if @panel.isVisible() then @panel.hide() else @panel.show()
 
+
+  #
+  # Invoked when the roku-develop:switch-files command is issued
+  #
+  switch: ->
+    editor = atom.workspace.getActivePaneItem()
+    panes = atom.workspace.getPanes()
+    if editor
```

Comment:
Need to check that editor.buffer exists otherwise editor.buffer.file will give TypeError exception.

```suggestion
    if editor and editor.buffer
```